wiki/Legal/Copyright/zh.md

```diff
@@ -0,0 +1,71 @@
+# 声明
+**这是一份osu!版权政策的非正式、非官方授权的中文翻译。它不是由osu!官方或ppy（Dean Herbert）发表，不具有任何法律效力。只有英文原版文本具有此等效力。**   
+**然而，我们希望这份翻译能帮助osu!的使用者更好地了解osu!的版权政策。对于因使用此翻译造成的任何问题，翻译者不承担任何法律责任。**
+
+# osu!版权政策
+
+考虑到版权侵犯的问题，osu!根据数字千年版权法案，美利坚合众国法典第17编512条采取如下政策。   
+
+osu!的指定接受侵权声明通知的代理人（“指定代理人”）的联系信息列在此政策最后，并记录在美利坚合众国版权办公室的文件中 (http://www.copyright.gov/onlinesp/list/index.html)。
+
+osu!十分严肃地对待版权和其它知识产权。
+
+
+osu!的政策将会（1）迅速阻止访问或移除根据诚信原则认为（believes in good faith）可能包含侵犯了第三方版权的材料并（2）移除持续侵权者并停止对他们的服务。
+
+
+
+# 举报版权侵犯
+如果你认为在存放在osu!网站或可由其访问的内容侵犯了您的版权，请向指定代理人发送包含以下信息的版权侵犯通知（联系法律顾问或参见美利坚合众国法典第17编512条(c)款来确认这些要求):
+
+1. 声明遭到版权侵犯之所有者授权采取行动的个人的实体或电子签名；
+
+2. 声明被侵权的作品之证明，或若单个通知中涉及统一网站上的多个侵权作品，此类作品的代表列表；
+
+3. 声明侵权或作为侵权行为主体之材料以及其是应被移除或禁止访问的内容的证明和足够使osu!定位此材料的信息；
+
+4. 申诉者的联系信息，如地址，电话号码，电子邮件（如果可用）；
+
+5. 声明申诉者根据诚信原则认为以被反馈之行动方法对材料的使用未被版权所有者、其代理人或法律授权的声明，和
+
+6. 声明通知内的信息准确，并且在知晓所言不实可受伪证罪处罚的情况下，通知的主体已经被授权以声明遭到版权侵犯之所有者之名义采取行动的声明。
+
+当一个完整和适当的版权侵犯通知被指定代理人接收，或osu!以其它方式根据诚信原则认为osu!服务上的一个文件可能包含侵犯版权的材料时：
+
+**根据我们的政策，osu!将会：**
+
+1. 移除或禁止访问在版权侵犯通知中被识别的内容；
+
+2. 通知内容提供者、会员和用户材料已经被移除或阻止访问；并且
+
+3. 在合适的条件下决定订阅者和账号拥有者是否为持续侵权者。
+
+## 提交反通知 (Counter-Notices)
+如果被移除或阻止访问之内容提供者、会员或用户相信其没有侵权，或内容提供者、会员或用户相信其已从版权拥有者、版权拥有者之代理人那里或依法得到授权有权发表并使用此内容时，内容提供者、会员或用户可以向指定代理人发送包含以下信息的反通知：
+
+1. 内容提供者、会员或用户的个人的实体的或电子的签名；
+
+2. 被移除或禁止访问的材料和其在被移除或禁止访问前位置的证明；
+
+3. 在知晓所言不实可受伪证罪处罚的情况下，内容提供者、会员或用户根据诚信原则认为被移除或禁止访问之内容是由于失误导致的结果或其是对被移除或禁止访问之内容的错误识别，以及：
+
+4. 内容提供者、会员或用户的姓名，地址，电话号码，以及电子邮件地址（如果可用）和此人或实体同意接受内容提供者、会员或用户所在地的美国联邦地方法院的司法管辖，或内容提供者、会员或用户的住址在美利坚合众国之外的声明；对于美国加利福尼亚州洛杉矶郡，需要同时提供此个人或实体会接受从提供了声称侵权的个人或此人之代理人处理服务的声明。
+
+若指定代理人收到反对通知书，osu!会向原始举报者发送一份反对通知书的副本以声明被移除的内容可能在10个工作日内被恢复或解除访问阻止。
+
+除非版权持有者采取行动以要求反对内容提供者、会员或用户之法院命令，被移除的内容在osu!独立自由裁量权下可能在收到反对通知书10到14个或更长的工作日后被恢复或解除访问阻止。
+
+## 指定代理人联系方式
+
+**电子邮件地址** ：[copyright@ppy.sh](mailto:copyright@ppy.sh) (保证24小时内回复)
+
+**邮寄地址** ：
+请参照英文版以获取澳大利亚邮政可识别的英文地址。
+```
+澳大利亚西澳大利亚州温布利格里高利街41号
```

Comment:
Here.
I am wondering whether English address is required.

Reply:
that would be better, since force locale(`?locale=en`) is not available any more ¯\_(ツ)_/¯

Reply:
Wait...
I cannot alter this branch anymore since it has already been merged to translate-zh.
Let's see if anyone with push access can change the merged branch.
Or should I just open another pull request?

Reply:
i add you to Translator team just now, feel free to edit on branch :p

Reply:
to be honest i'm not even sure if this page should be translated in the first place.

Reply:
I once had the same doubt, sir, and that is why I followed Chinese osu! translation community's suggestion and added a disclaimer, translation below:

```
This is a unoffical Chinese translation of osu! copyright policy without offical authorization. It is released neither by osu! nor by ppy(Dean Herbert), which has no legal validity. Only original version in English has such validity.

However, we hope this translation helps osu! users(those who cannot fully understand DMCA and policies) to have a better understanding of osu! copyright policies. Translators shall assume no legal responsibilities related to any issue caused by using this translation.
```

The main purpose of this translation is to let some Chinese user understand osu! policies - bacause a very little small amount of them doesn't care such policies and use "I don't understand English" as a reason... And they sometimes cause harm to organizations we love such as osu! lol  

Admittedly, I, until now, still cannot ensure that this translation is okay if somebody directly use it, but I think such translation maybe okay and useful when I see the Russian translation of this policy(and a translation of GPLv2, which claims that the only good to use one is the original one). 

;->